index.html

```diff
@@ -2,15 +2,55 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>HYF</title>
+    <title>Hack Your Future Belgium</title>
     <meta name="description" content="HYF webpage" />
     <meta name="author" content="HYF-GROUP2" />
+    <link rel="preconnect" href="https://fonts.gstatic.com"> 
```

Comment:
On the site the fonts are SpaceMono and WorkSans… 
You want to change it to Roboto?😊

Reply:
i didnt notice that i changed them to Roboto :) i will change